core/taskengine/executor.go

```diff
@@ -81,13 +81,7 @@ func (x *TaskExecutor) RunTask(task *model.Task, queueData *QueueExecutionData)
 		return nil, fmt.Errorf("internal error: invalid execution id")
 	}
 
-	triggerMetadata := queueData.Reason
-	if triggerMetadata == nil {
-		x.logger.Debug("Task trigger reason is nil, creating default reason", "task_id", task.Id, "execution_id", queueData.ExecutionID)
-		triggerMetadata = &avsproto.TriggerReason{
-			Type: avsproto.TriggerReason_Manual,
-		}
-	}
+	triggerMetadata := GetTriggerReasonOrDefault(queueData.Reason, task.Id, x.logger)
```

Comment:
The function `GetTriggerReasonOrDefault` is to return a triggerReason. Does this variable `triggerMetadata` mean trigger reason? If so, please rename the variable.